queries.sql

```diff
@@ -44,3 +44,13 @@ SELECT species.name, COUNT(animals.name) FROM animals JOIN species ON species.id
 SELECT animals.name FROM animals JOIN species ON species.id = animals.species_id JOIN owners ON owners.id = animals.owner_id WHERE owners.full_name = 'Jenniffer Orwell' AND species.name = 'Digimon';
 SELECT animals.name FROM animals JOIN owners ON owners.id = animals.owner_id WHERE animals.escape_attempts = 0 AND owners.full_name = 'Dean Winchester';
 SELECT COUNT(owners.full_name), owners.full_name FROM animals JOIN owners ON owners.id = animals.owner_id GROUP BY owners.full_name ORDER BY COUNT(owners.full_name) DESC;
+
+SELECT animals.name, visits.visit_date FROM animals JOIN visits ON visits.animal_id = animals.id JOIN vets ON vets.id = visits.vet_id WHERE vets.name = 'William Tacher' ORDER BY visits.visit_date ASC LIMIT 1;
+SELECT animals.name, visits.visit_date FROM animals JOIN visits ON visits.animal_id = animals.id JOIN vets ON vets.id = visits.vet_id WHERE vets.name = 'Stephanie Mendez' GROUP BY (visits.visit_date, animals.name);
+SELECT vets.name, species.name FROM vets LEFT JOIN specialization ON specialization.vet_id = vets.id LEFT JOIN species ON species.id = specialization.species_id;
+SELECT animals.name, visits.visit_date FROM animals JOIN visits ON visits.animal_id = animals.id JOIN vets ON vets.id = visits.vet_id WHERE visits.visit_date BETWEEN '2020-4-1' AND '2020-8-30';
```

Comment:
- [ ] It would be better if we are able to make this query -> `List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020`. Let's take a closer look at it very well